invenio/base/scripts/demosite.py

```diff
@@ -94,10 +96,9 @@ def populate(packages=[], default_data=True, files=None,
     from werkzeug.utils import import_string
     map(import_string, packages)
 
-    from invenio.ext.sqlalchemy import db
     print(">>> Going to load demo records...")
-    db.session.execute("TRUNCATE schTASK")
```

Comment:
`TRUNCATE` was resetting `autoincrement` column. Can we do the same or use the low-level task submission?